NCATS_Data_Dictionary.md

```diff
@@ -52,82 +52,82 @@ This information is organized by database and collection (table).
 The data in this collection is derived from the National Vulnerability
 Database [CVE feeds](https://nvd.nist.gov/vuln/data-feeds).
 
-- `_id` [string]: [Common Vulnerabilities and
+- `_id` [string]:[Common Vulnerabilities and
```

Comment:
Somehow you managed to remove all of the spaces after the colons in the upper portion of the data dictionary (everything outside of the `assessment` database).  Please return our missing spaces and no further questions will be asked.

Reply:
They're being held in an undisclosed location until our demands are met.